lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -98,6 +98,13 @@ aaa_server_group:
   vrf:
     default_value: "default"
 
+dnsclient:
+  name_server:
+    # NOTE: This does not account for vrf ip name-server
+    config_get: 'show hosts'
```

Comment:
When possible I would prefer we use the running-config instead of a feature show command.

# show running-config all | include 'ip name-server'
ip name-server 5.5.5.5 7.7.7.7 55:44::3
#